x-pack/plugins/reporting/server/core.ts

```diff
@@ -357,30 +364,44 @@ export class ReportingCore {
     return startDeps.esClient;
   }
 
-  public getScreenshots(options: ScreenshotOptions): Rx.Observable<ScreenshotResult> {
+  private mapToScreenshotConfigOptions(options: PdfScreenshotOptions | PngScreenshotOptions) {
+    const config = this.getConfig();
+    return {
+      timeouts: {
+        loadDelay: durationToNumber(config.get('capture', 'loadDelay')),
+        openUrl: durationToNumber(config.get('capture', 'timeouts', 'openUrl')),
+        waitForElements: durationToNumber(config.get('capture', 'timeouts', 'waitForElements')),
+        renderComplete: durationToNumber(config.get('capture', 'timeouts', 'renderComplete')),
```

Comment:
I believe we should create an issue to  move all of the `capture` config properties out of Reporting and into Screenshotting. I don't think different consumers of Screenshotting will need to use different options - WDYT?

Reply:
I like this idea! https://github.com/elastic/kibana/issues/127750